### PR TITLE
Added getrgb HSB/HSV color strings

### LINKED ISSUE #2118: PIL.ImageColor.getrgb add support for HSB/HSV format
The current `PIL.ImageColor.getrgb` does not support [HSB/HSV](https://en.wikipedia.org/wiki/HSL_and_HSV) format, only the related HSL, but for easier conversion of some coordinates it would be nice to have support for HSB/HSV also.

### PR DESCRIPTION
**Pull request by radarhere**
Resolves #2118

### CODE REVIEW

File: src/PIL/ImageColor.py [hunk 1]
```diff
@@ -101,6 +101,20 @@ def getrgb(color):
             int(rgb[2] * 255 + 0.5)
             )
 
+    m = re.match(r"hs[bv]\(\s*(\d+\.?\d*)\s*,\s*(\d+\.?\d*)%\s*,\s*(\d+\.?\d*)%\s*\)$", color)
```
Comment (hugovk) [inline src/PIL/ImageColor.py]: Here, and in others, looks like the initial opening, and final closing brackets are also redundant.

Shall we remove these to make the regex a bit shorter? The `m.group` calls can then begin from 0.

File: docs/reference/ImageColor.rst [hunk 2]
```diff
@@ -31,6 +31,13 @@ The ImageColor module supports the following string formats:
   (black=0%, normal=50%, white=100%). For example, ``hsl(0,100%,50%)`` is pure
   red.
 
+* Hue-Saturation-Value (HSV) functions, given as ``hsv(hue, saturation%,
+  value%)`` where hue and saturation are the same as HSL, and value is a value
+  between 0% and 100% (black=0%, normal=100%). For example,
```
[the remark this answers is not in the document]
Comment (radarhere) [reply docs/reference/ImageColor.rst]: Okay, updated.

File: src/PIL/ImageColor.py (hunk 1 above)
Comment (radarhere) [reply src/PIL/ImageColor.py]: The initial and final closing brackets are escaped - they are part of the string, e.g. `hsb(0,100%,50%)`

Comment (hugovk) [reply src/PIL/ImageColor.py]: Haha, we'd better keep them then!

**Pull request merged**